pyiron/atomistics/job/atomistic.py

```diff
@@ -10,6 +10,7 @@
 
 from pyiron.atomistics.structure.atoms import Atoms
 from pyiron_base import GenericParameters, GenericMaster, GenericJob as GenericJobCore
+from pyiron_base.generic.util import deprecate
```

Comment:
I would like to import it directly from `pyiron_base` this would allow us to change the structure in `pyiron_base` independent of changing `pyiron` and at the same time we can make sure that only the top level functions are used in other modules.

Reply:
Addressed in https://github.com/pyiron/pyiron_base/pull/122

Reply:
Fill fix once we update the version of pyiron_base.